lexicons/app/bsky/actor/defs.json

```diff
@@ -80,6 +80,7 @@
       "type": "object",
       "properties": {
         "muted": {"type": "boolean"},
+        "mutedByList": {"type": "ref", "ref": "app.bsky.graph.defs#listViewBasic"},
```

Comment:
I assume this will be used to give viewers context about why the content was muted, does that sound right?

This _could_ be an array, but fine by me if we just want to surface one list responsible for the blocking.

Reply:
yup exactly

I thought about doing an array but just decided to do the one. this could end up getting arbitrarily large if we make it an array. & i imagine the UI will only ever show one reason at a time